demo/kubernetes/rook-client/rook-client.yml

```diff
@@ -2,17 +2,15 @@ apiVersion: v1
 kind: Pod
 metadata:
   name: rook-client
+  namespace: rook
 spec:
   containers:
   - name: rook-client
-    image: quay.io/rook/k8s-demo
+    image: quay.io/rook/rook-client
     imagePullPolicy: IfNotPresent
     command: ["sleep", "36500d"]
     securityContext:
       privileged: true
-    env:
-      - name: ROOK_API_SERVER_ENDPOINT
```

Comment:
k8s inserts environment variables for the service hosts and endpoints. Since the REST api is exposed as a k8s service, the pods will get these environment variables included:
```
ROOK_REST_API_SERVICE_HOST=10.3.0.125
ROOK_REST_API_SERVICE_PORT=8124
```